post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+:context: creating-multi-arch-compute-nodes-ibm-power
+[id="creating-multi-arch-compute-nodes-ibm-power"]
+= Creating a cluster with multi-architecture compute machines on {ibmpowerProductName}
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+To create a cluster with multi-architecture compute machines on {ibmpowerProductName}(`ppc64le`), you must have an existing single-architecture (`x86_64`) cluster. You can then add `ppc64le` compute machines to your {product-title} cluster.
+
+Before you can add `ppc64le` nodes to your cluster, you must upgrade your cluster to one that uses the multi-architecture payload. For more information on migrating to the multi-architecture payload, see xref:../../updating/updating_a_cluster/migrating-to-multi-payload.adoc#migrating-to-multi-payload[Migrating to a cluster with multi-architecture compute machines].
+
+The following procedures explain how to create a {op-system} compute machine using a {op-system-base} KVM instance. This will allow you to add `ppc64le` nodes to your cluster and deploy a cluster with multi-architecture compute machines.
+
```

Comment:
Should we mention any gotcha's here? Or is that better for a technote? 

Otherwise LGTM

Reply:
Thanks @prb112 
Updated the PR with a note. Request you to check. Thanks.

Reply:
@alishaIBM :  Can you remove the reference to KVM?

Reply:
@manojnkumar I have removed the reference to KVM. 
Is it okay to write: "The following procedures explain how to create a RHCOS compute machine using an ISO image or network PXE booting."
Please confirm. Thanks.

Reply:
That's fine.